src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs

```diff
@@ -123,6 +127,10 @@ public override void ExecuteCmdlet()
             {
                 databaseAccountUpdateParameters.DisableKeyBasedMetadataWriteAccess = DisableKeyBasedMetadataWriteAccess;
             }
+            if (DisableKeyBasedMetadataWriteAccess != null)
```

Comment:
if (PublicNetworkAccess != null)

Reply:
I actually caught the typo beforehand and fixed it with it2